### DISCUSSION TITLE
BlackHole not sending Garageband Audio to Zoom on Apple M1 Pro Ventura 13.5.2

### DISCUSSION THREAD

**Discussion by kelcydw** [ExistentialAudio/BlackHole#790] [Q&A]
Trying to do a Zoom livestream using audio from Garageband (via Scarlett18i20) and have used BlackHole successfully on my 2017 iMac running Catalina 10.15.7—but this 2017 iMac is on the outs. I tried setting up everything the same way on the newer M1 Macbook Pro w/ Ventura 13.5.2, and the Zoom microphone test appears to receive the audio but playback is distorted. I have tried everything I've been able to find online and can't find a fix. 

In Audio MIDI Setup I've created the multi-output with MacBook Speakers (1st), BlackHole 16ch (Drift correction on), and Scarlett 18i20 USB (drift correction on). This output is selected as the MacBook system output. 

In Garageband I have two tracks set up with their corresponding Scarlett inputs, recording enabled, and monitoring on for both tracks. Audio sounds as it should. Output device is the multi-output and input device is the Scarlett. 

In Zoom (updated today) I have Speaker set to Same as System (multi-output), but I have changed this to every other speaker and the result doesn't change. I have the microphone set to BlackHole 16ch and when I test the mic, the audio does come from Garageband but is garbled/distorted. When I switch microphone to Scarlett, the test mic records the audio cleanly.

I have compared everything between the two computers, and I can't see why it works fine on the old computer and not on the new computer running Ventura. The Scarlett connects to the MacBook Pro with a hub, not a direct USB C connection—could that be a problem? 

Appreciate any advice! I need to be able to Zoom livestream w/ Garageband-treated audio from this MacBook Pro. Thanks

**Comment by devinroth** [comment]
Probably have background sound cancellation enabled on Zoom. Be sure to enable original audio for musicians and enable stereo. Look at zoom documentation if you have any questions about that.

Devin
